test/fixed-point.jl

```diff
@@ -313,19 +323,22 @@ end
         @test sprint(showcompact, FD2(-42.40)) == "-42.4"
         @test sprint(showcompact, FD2(-0.01)) == "-0.01"
         @test sprint(showcompact, FD2(0)) == "0.0"
-        for x in keyvalues[FD2]
-            if 0 ≤ abs(x) < 1000
-                @test eval(parse(string(x))) == x
-            end
-        end
 
-        @test string(typemin(FixedDecimal{Int64, 2})) ==
+        @test sprint(show, typemin(FixedDecimal{Int64, 2})) ==
```

Comment:
similarly, instead of `sprint(show` we can do `repr` which should be equivalent here.